src/app/retrospective/retrospective-create/retrospective-create.component.scss

```diff
@@ -1,10 +1,16 @@
+mat-card {
```

Comment:
Set min width for each card so that when we resize the window, width of each card remains the same and a horizontal/vertical scroll is visible inside the modal

Reply:
Added min-width css attribute to the parent "mat-grid-list" tag to handle the required use-case